Docker_cram/docker-compose.yml

```diff
@@ -1,14 +1,27 @@
+# version: '3'
+# services:  
+# # the giskard container
+#   cram:
+# #    image: "bhavyawadhwa/cram:testing"
+# #    container_name: container_cram
+#     image: "cram_image"
+#     container_name: cram_container
+#     network_mode: host
+#     privileged: true
+#     environment:
+#       - DISPLAY=${DISPLAY}
+#     volumes:
+#       - "$HOME/.Xauthority:/root/.Xauthority:rw"
+#     network_mode: host
 version: '3'
 services:  
-# the giskard container
   cram:
-    image: "bhavyawadhwa/cram:testing"
-    container_name: container_cram
+    image: "arthurniedz/cram:latest"
```

Comment:
let's keep it with the tag latest, and comit changes to it...Also important thing to keep in mind, some steps:-
1. Change docker file and .sh file as you want 
2. build image 
3. if you again change .sh file, again you need to build image because otherwise it will not alter those updates.
4. once everything is done then pull on docker hub